import-scripts/oncokb-annotator.sh

```diff
@@ -311,12 +307,30 @@ FLOCK_FILEPATH="/data/portal-cron/cron-lock/oncokb-annotator.sh"
         gzip_oncokb_annotated_files
         echo $(date)
         echo "Running rsync to download.cbioportal.org for OncoKB Annotation for MSKSOLIDHEME"
-        rsync_oncokb_annotated_files
+        rsync --exclude "/data/portal-cron/git-repos/oncokb-annotated-msk-impact/.git" -a $ONCOKB_OUT_DIR/ $ONCOKB_ANNOTATED_GIT_DIR
     else
         message="OncoKB Annotation failed for MSKSOLIDHEME. Check logs for more details."
         send_failure_messages "$message" "MSKSOLIDHEME OncoKB Annotation Failure" "MSKSOLIDHEME OncoKB Annotation"
     fi
 
+    # Create and push a git amended changeset
+    cd $ONCOKB_ANNOTATED_GIT_DIR    
```

Comment:
make this commit & push dependent on annotation success

Reply:
Done - this step was moved into the if-then conditional block. To reduce nesting, the add/commit/push steps were put into a named function call, and all three steps (gzip, rsync, git commit/push) were standardized in format.